WooCommerce/Classes/Authentication/Epilogue/StorePickerViewController.swift

```diff
@@ -116,6 +116,14 @@ class StorePickerViewController: UIViewController {
         super.viewWillDisappear(animated)
         onDismiss?()
     }
+
+    override func viewDidDisappear(_ animated: Bool) {
+        super.viewDidDisappear(animated)
+
+        // This should be called here to address this issue:
+        // https://github.com/woocommerce/woocommerce-ios/issues/693
+        stopListeningToNotifications()
```

Comment:
If this is here, should the `deinit` method be deleted? 

I can't remember if it's remove observer called twice or add observer called twice that can cause crashes.

Reply:
Yeah, this for me — I'll remove it.